### DISCUSSION TITLE
Add Overture Themes - Places, Divisions, Transportation, Buildings, Base (natural features/other), and Addresses

### DISCUSSION THREAD

**Discussion by larryfreil** [microsoft/PlanetaryComputer#391] [Data request]
I would like to recommend adding a copy/mirror of the overture datasets to the Planetary computer catalog. The different themes could be useful in a variety of approaches, and potentially could be useful for some of the other data asks. For example the 'Base' theme includes a lot of natural features and landcover-like features, the buildings theme includes the MS buildings mixed and conflated with OSM, Google open buildings, and ESRI contributed building, and Divisions includes a lot of the administrative boundaries around the world, to name a few.

**Comment by TaylorCorbett** [comment]
Thanks for this recommendation Larry! I'll follow up with you separately to discuss the possibility of onboarding a copy of the Overture datasets into the Planetary Computer.
